src/api/idea-card/content-types/idea-card/lifecycles.js

```diff
@@ -8,19 +8,62 @@ module.exports = {
   },
 
   async afterCreate(event) {
-    const { id: id, ideaName: ideaName, createdAt: timeCreated } = event.result;
+    const { id: id, ideaName: ideaName, createdAt: timeCreated, tagline: tagline, author: author } = event.result;
     
-    await strapi.entityService.create('api::notification.notification', {
+    await strapi.entityService.create('api::event.event', {
       data: {
-        Title:"A new idea has been created",
-        Content: "You created idea " + ideaName,
+        Title:"Idea Submitted Successfully",
+        content: author + " added new idea, " + ideaName + "-" + tagline +  "is created",
         entityType: "IdeaCard",
         entityId: id,
-        TimeCreated: timeCreated,
-        Read: false
+        createdDateTime: timeCreated
       },
     });
-  }
+  },
+
+  async beforeUpdate(event) {
+    const { data, where, select, populate } = event.params;
+    const previousData = await strapi.entityService.findOne('api::idea-card.idea-card', where.id, { populate });
+
+    // Store the previous status in the context
+    event.params.data.previousStatus = previousData.status;
+  },
+
+  async afterUpdate(event) {
+    const { result, params } = event;
+    const { previousStatus } = params.data;
+
+    if (result.status !== previousStatus) {
+      
+      const { id, ideaName, status, createdAt } = result;
```

Comment:
Can we extract this block into a separate method to avoid duplication or simplify to create `title` and `content` conditionally and use them in one call?